package.json

```diff
@@ -7,15 +7,18 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "src/index.js",
+  "main": "ui-kit.cjs.js",
+  "module": "ui-kit.esm.js",
```

Comment:
💯

Reply:
In node land ESM modules (if my memory is still correct) can only be supported by the `mjs` extension. Should we go for the same extension here?

Reply:
Ah, I also still wanted to look into the difference of specifying `es` and `esm` as the output.